DependencyInjection/Compiler/NotifierCompilerPass.php

```diff
@@ -58,18 +57,10 @@ public function process(ContainerBuilder $container)
                 // Define Notification form
                 $notificationFormAlias = sprintf('notification_%s', $alias);
                 $notificationFormServiceId = sprintf('idci_notification.form.type.%s', $notificationFormAlias);
-                $notificationFormDefinition = new DefinitionDecorator('idci_notification.form.type.abstract_notification');
-                $notificationFormDefinition->replaceArgument(0, $alias);
-                $notificationFormDefinition->replaceArgument(1, $notifierReference);
-                $container->setDefinition($notificationFormServiceId, $notificationFormDefinition);
```

Comment:
Why

Reply:
i have remove AbstractNotificationType & AbstractNotifierType class and add a event listener on PRE_SET_DATA for manage field with type selected

https://github.com/Tessi-Tms/NotificationBundle/blob/master/Form/NotificationType.php#L42-L75
https://github.com/Tessi-Tms/NotificationBundle/blob/master/Form/NotifierConfigurationType.php#L31-L44

with sf3.0 on createForm i need to call type with "::class" your abstract containt a construct but is not callable.